src/js/components/MenuCategory.js

```diff
@@ -0,0 +1,56 @@
+import { CATEGORY_LIST } from '../constant/index.js';
+import Component from './Component.js';
+
+export default class MenuCategory extends Component {
+  constructor(containerId) {
+    const menuGroupTemplate = `<button
+    id="{{id}}"
+    class="cafe-category-name btn bg-white shadow mx-1"
+  >
+    {{name}}
+  </button>`;
+    super(containerId, menuGroupTemplate);
+  }
+
+  init() {
+    window.location.hash = 'espresso';
+
+    this.container.addEventListener('click', ({ target }) => {
+      if (!target) return;
+
+      if (target.classList.contains('cafe-category-name')) {
+        const categoryId = target.id;
+        const categoryName = target.textContent;
```

Comment:
```suggestion
        const { id: categoryId, textContent: categoryName } = target;
```
이렇게 구조분해할당을 사용 할 수도 있겠어요~!